src/storage/storage.cc

```diff
@@ -51,7 +51,13 @@ class StorageImpl : public Storage {
   static void ActivateDevice(Context ctx) {
```

Comment:
Seems this method does not need to be static. It is only used in this class. Besides, making it static without a RAII class around is very dangerous.

Reply:
You raise a good point, but I think that should be addressed in a separate PR.